### DISCUSSION TITLE
docs/rbac-with-domains-api

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1254] [Docs comments]
# docs/rbac-with-domains-api

A more friendly API for RBAC with domains. This API is a subset of Management API. The RBAC users could use this API to simplify the code

https://casbin.org/docs/rbac-with-domains-api

**Comment by umairrsyedd** [comment]
Unable to find the GetImplicitUsersForResourceByDomain() function, Using the gorm adapter with golang.

**Reply by PokIsemaine** [reply]
Thanks for your feedback, this API was recently added by Casbin and has not yet been extended to the gorm adapter. We will finish it in the near future

**Reply by umairrsyedd** [reply]
Okay, Thanks for the update @PokIsemaine

**Reply by hsluoyz** [reply]
Tracked here: https://github.com/casbin/gorm-adapter/issues/216

**Comment by GauravDubey56** [comment]
Is there a GetPermissionsForUserInDomain function in Node js library

**Reply by hsluoyz** [reply]
@GauravDubey56 you can create a github issue at Node-Casbin repo to request this feature

**Reply by GauravDubey56** [reply]
can I create the issue and work on it myself? I have been using Casbin js at my org and would like to implement this

**Reply by hsluoyz** [reply]
@GauravDubey56 yes, issue and PR are welcome!
